scylla/datadog_checks/scylla/data/conf.yaml.example

```
@@ -0,0 +1,152 @@
+init_config:
+
+instances:
+
+    ## @param instance_endpoint - string - optional
+    ## The URL where your database metrics are exposed by Prometheus.
+    ## By default, the Scylla integration collects `instance_endpoint` metrics.
+    ## At least one of instance_endpoint or manager_endpoint must be provided.
+    #
+  - instance_endpoint: http://localhost:9180/metrics
+
+    ## @param metric_groups - list of strings - optional
+    ## List of additional scylla metric groups to collect in addition to the default
+    ## These are only applicable to instance_endpoints, manager_endpoints collect all metrics
```

```suggestion
    ## These are only applicable when instance_endpoint is configured. manager_endpoint collect all metrics.
```
nit: pluralizing the config option is a bit confusing to read.

---

👍 In fact, will remove that line since we no longer have manager covered (missed those lines in this conf)